enos/provider/g5k.py

```diff
@@ -394,37 +404,71 @@ def pick_nodes(pool, n):
             nodes = pool[:n]
             del pool[:n]
             return nodes
+        
+        def mk_roles(pools, resources):
+            """
+            Distribute the nodes of pools according to the resources claim
+            resources : 
+                cluster1: 
+                    role11: n11
+                    role12: n12
+                    ...
+                cluster2:
+                    role21: n21
+                    role22: n22
+            pools:
+                cluster1: [node11, node12, ...]
+                cluster2: [node21, node22, ...]
+
+            expected output
+
+            roles:
+                role'1: [node'11, node'12 ...]
+                role'2: [node'21, node'22 ...]
+            """
```

Comment:
Could you go with concrete names in the documentation? I don't understand why the value of `role'1` is `[node'11, node'12 ...]`